.env-example

```diff
@@ -19,6 +19,8 @@ LOG_FORMAT=dev
 # removeURL=https://apps.canada.ca/x-notify/subs/remove/
 #
 
+COMPLETED_JOBS_TO_KEEP=300 #Max number of completed jobs to keep in queue.  Older jobs are automatically purged.
```

Comment:
Add this to the docs for env variable here: https://github.com/ServiceCanada/x-notify/blob/master/setup.md#environment-variables